lib/helper_method.rb

```diff
@@ -0,0 +1,110 @@
+module Hotel
+  class Helper_Method
+
+### SORTING METHODS ###
+    def self.sort_reservations_by_date(list_of_reservations)
+      list_of_reservations.sort! { |a,b| a.check_in_date <=> b.check_in_date }
+    end
+
+    def self.sort_block_reservations_by_name(list_of_block_reservations)
+      # SORTS AN ARRAY OF ARRAYS BY NAME
+      list_of_block_reservations.sort! { |a,b| a.first.block_name <=> b.first.block_name }
+    end
+
+    def self.connect_reservation_to_room_and_sort(list_of_rooms, room_number, reservation)
+      connected_room_number = Hotel::Helper_Method.find_room_number(list_of_rooms, room_number)
```

Comment:
By the way, Ruby will always check for local names first, so there's no need to namespace fully... this should still work even if it's just `connected_room_number = find_room_number(list_of_rooms, room_number)`